app/views/recipes/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<h1 class="mt-4 text-center">Add new Recipe</h1>
+<div class="row mt-5">
+  <div class="col-md-6 offset-md-3">
+    <div class="card">
+      <div class="card-body">
+        <%= form_with model: @recipe, url: user_recipes_path, class: 'flex flex-col items-center w-6/12 mx-auto' do |form| %>
+          
+
+          <div class="input-group input-group-outline mb-3 row">
+            <label class="form-label col-sm-3">Name</label>
+            <%= form.text_field :name, class: 'form-control focus:border-blue-600 col-sm-9' %>
+          </div>
+
+          <div class="input-group input-group-outline mb-3 row">
+            <label class="form-label col-sm-3">Preparation time</label>
+            <%= form.number_field :preparation_time, class: 'form-control focus:border-blue-600 col-sm-9' %>
+          </div>
+
+          <div class="input-group input-group-outline mb-3 row">
+            <label class="form-label col-sm-3">Cooking time</label>
+            <%= form.number_field :cooking_time, class: 'form-control focus:border-blue-600 col-sm-9' %>
+          </div>
+
+          <div class="input-group input-group-outline mb-4 row">
+            <label class="form-label col-sm-3">Description</label>
+            <%= form.text_area :description, class: 'form-control focus:border-blue-600 col-sm-9' %>
+          </div>
+
+          <div class="input-group input-group-outline mb-4 row">
+            <label class="form-label col-sm-3">Public</label>
+            <%= form.check_box :public, class: 'form-check-input col-sm-9' %>
+          </div>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/219689070-eb144f96-aa3c-4ded-8cf0-ab039b4ee315.png)

**[Optional]**

- [ ] When i tried to create a recipe without making it public then it stayed on this page means it was not allowing me to make my recipe private :fearful: Kindly fix this issue to make your project more professional and human friendly :+1:

Reply:
Done